CMS-Enterprise/easi-app
# [EASI-4662] Add un-retire LCID to IT gov admin actions

# EASI-4662

## Description

- Add new admin action that allows ITGO admins to un-retire a previously retired LCID. This basically just nulls out the lcidRetiresAt field
- Add new DB migration and graphql types/resolvers to handle new admin action
- Add emails that are sent to the requester informing them of the un-retirement of their LCID

## How to test this change

- Ensure that new "remove LCID expiration" action only appears on LCIDs that are already retired
- Retire an LCID and ensure that the new action appears as intended
- Remove the retirement date from an LCID and ensure that lcidRetiresAt is null while all other related fields are untouched
- Ensure emails are correctly generated and sent

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
